### PR TITLE
update CDI API to 2.0.SP1

### LINKED ISSUE #13653: Reconsume CDI 2.0.SP1
We need to move up to the latest Jakarta EE 8 versions of the CDI API and Weld Impl...

https://repo1.maven.org/maven2/jakarta/enterprise/jakarta.enterprise.cdi-api/2.0.2/jakarta.enterprise.cdi-api-2.0.2.jar
https://repo1.maven.org/maven2/org/jboss/weld/weld-osgi-bundle/3.1.5.Final/weld-osgi-bundle-3.1.5.Final.jar

Notice that the API is a Jakarta one and so we need to make sure that things like the maven co-ords are correct. It is also possible there may be a knock on to other Jakarta EE 8 APIs that also need updating??

### PR DESCRIPTION
**Pull request by benjamin-confino**
fixes #13653

### CODE REVIEW

**Comment by benjamin-confino** [comment]
build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

File: dev/com.ibm.websphere.javaee.cdi.2.0/bnd.bnd [hunk 1]
```diff
@@ -54,5 +54,5 @@ publish.wlp.jar.suffix: dev/api/spec
 
 -buildpath: \
 	javax.inject:javax.inject;version=1,\
-	javax.enterprise:cdi-api;version=2.0
-	
\ No newline at end of file
+	javax.enterprise:cdi-api;version=2.0.0.SP1
```
Comment (tevans78) [inline dev/com.ibm.websphere.javaee.cdi.2.0/bnd.bnd:57]: Not sure how much of a difference it makes but the original only has one zero
```suggestion
	javax.enterprise:cdi-api;version=2.0.SP1
```

**Review by tevans78 — APPROVED** [review]

File: dev/com.ibm.websphere.javaee.cdi.2.0/bnd.bnd (hunk 1 above)
Comment (benjamin-confino) [reply dev/com.ibm.websphere.javaee.cdi.2.0/bnd.bnd:57]: This is to comply with bndtools syntax rules

**Comment by benjamin-confino** [comment]
#run-libby-bot

**Pull request merged**